### ISSUE TITLE
Resolution of training data

### ISSUE THREAD

**Issue by lianqiyan** [mengab/MGANet#1]
I just wonder if you select the video sequences which are in the same resolution? Or the training set are consist of different resolution videos?

**Comment by mengab** [comment]
> I just wonder if you select the video sequences which are in the same resolution? Or the training set are consist of different resolution videos?


The video sequences included in the training set have different resolutions, i.e., CIF, 240p, 360p, 480p, 720p and 1080p, etc.

**Comment by lianqiyan** [comment]
> > I just wonder if you select the video sequences which are in the same resolution? Or the training set are consist of different resolution videos?
> 
> The video sequences included in the training set have different resolutions, i.e., CIF, 240p, 360p, 480p, 720p and 1080p, etc.

By the way, how can I download the 18 sequences of Classes A-E with different resolutions from the Joint Collaborative Team on Video Coding (JCT-VC) standard test set, I have searched for it many times but still can't find the resource

**Comment by AlexSun0110** [comment]
您好，我是最近正在做压缩视频质量增强。看到您似乎也在做这个研究，不知道可否加下联系方式进行些交流呢。十分感谢！我的邮箱：alexsun0110@qq.com

> > > 我只是想知道您是否选择了相同分辨率的视频序列？还是训练集由不同分辨率的视频组成？
> > 
> > 
> > 训练集中包含的视频序列具有不同的分辨率，即CIF，240p，360p，480p，720p和1080p等。
> 
> 顺便说一句，我如何从视频编码联合协作小组（JCT-VC）标准测试集下载分辨率不同的AE类的18个序列，我已经搜索了很多次，但仍然找不到资源

您好，我是最近正在做压缩视频质量增强。看到您似乎也在做这个研究，不知道可否加下联系方式进行些交流呢。十分感谢！我的邮箱：alexsun0110@qq.com

**Comment by AlexSun0110** [comment]
> > > 我只是想知道您是否选择了相同分辨率的视频序列？还是训练集由不同分辨率的视频组成？
> > 
> > 
> > 训练集中包含的视频序列具有不同的分辨率，即CIF，240p，360p，480p，720p和1080p等。
> 
> 顺便说一句，我如何从视频编码联合协作小组（JCT-VC）标准测试集下载分辨率不同的AE类的18个序列，我已经搜索了很多次，但仍然找不到资源

您好，我是最近正在做压缩视频质量增强。看到您似乎也在做这个研究，不知道可否加下联系方式进行些交流呢。十分感谢！我的邮箱：alexsun0110@qq.com
